_guides/reverse-proxies.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: documentation
+title: TBD
```

Comment:
I couldn't make up my mind on a decent title. I came up with the following but nothing really clicks, so feedback welcome and then I'll amend this PR with something:


- Configuring common web servers to proxy The Lounge
- Deploying The Lounge using common web servers
- Setting up common web servers as proxy
- The Lounge and web servers
- Configuring a web server to proxy The Lounge
- Running The Lounge using a web server
- Configuring a reverse proxy for The Lounge

Reply:
`Configure a reverse proxy for The Lounge` I like that.